src/pages/[platform]/build-a-backend/auth/auth-migration-guide/index.mdx

```diff
@@ -1806,33 +1687,14 @@ This API has been deprecated: existing use cases can be migrated to the `confirm
     }
     ```
 
-  </div>
-  <div>
-    **V6**
-    
-    ```
-    import { confirmSignIn } from 'aws-amplify/auth';
-
-    const handleConfirmSignIn = async ({ challengeResponse }) => {
-      const {
-        isSignedIn,
-        nextStep
-      } = await confirmSignIn({ challengeResponse });
-    }
-    ```
-
-  </div>
-</Columns>
+  </Block>
+</BlockSwitcher>
 
 ## Auth.getPreferredMFA
 
-```diff
--  import { Auth } from 'aws-amplify';
-+  import { fetchMFAPreference } from 'aws-amplify/auth';
+In v6, `Auth.getPreferredMFA` has become the `fetchMFAPreference` API. This API no longer accepts a `CognitoUser`: all Auth API’s now use the current signed in user information to perform operations, so sending and receiving `CognitoUser` through every API is no longer required. There is also no longer a bypassCache option in v6, as `fetchMFAPreference` will always makes a call to the server.
 
--  Auth.getPreferredMFA: (user: CognitoUser | any, params?: GetPreferredMFAOpts) => Promise<string>;
-+  fetchMFAPreference: () => Promise<FetchMFAPreferenceOutput>;
-```
+The return value in v6 is no longer just a string, but is now an object containing an `enabled` array and a `preferred` string.
```

Comment:
probably expanding more what `enabled` and `preferred` mean here. 

Cognito can have multiple `enabled` MFA options as of now, it only supports `TOTP` and `SMS`. An MFA option is considered to be enabled when it was previously setup. For instance, to set up `TOTP`, it can be done during the `sign-in` process when the `signIn` API returns ` the 'CONTINUE_SIGN_IN_WITH_TOTP_SETUP' step. It can also be setup when the user is authenticated by subsequently calling `setupTOTP` and `verifyTOTPSetup`.

On the other hand, to set up `SMS` the user needed to provide a `phone_number` during sign-up

Reply:
In addition, Cognito only allows 1 option to be `preferred`, this can be achieved my calling the `updateMFAPreference` API

Reply:
Love this, was definitely running out of steam here, this content is very helpful.

Reply:
+2